.github/workflows/main.yml

```diff
@@ -0,0 +1,33 @@
+name: Build
+on: [push,pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
Do you think we could run another build using ubuntu `16.04`?

Reply:
Done https://github.com/BugSwarm/bugswarm/pull/11/commits/719cd0d42502d7b51734b6c07453aafbfe58a102 and https://github.com/dtomassi/bugswarm/commit/719cd0d42502d7b51734b6c07453aafbfe58a102/checks?check_suite_id=398252524